app/page.tsx

```diff
@@ -52,23 +51,36 @@ export default async function Home({
 						/>
 						Passport Data Pages
 					</h1>
-					<div className="rounded-sm border-[3px] border-amber-400 flex flex-col justify-center w-full md:w-10/12 gap-2 p-3 sm:p-4 my-4 mx-auto break-inside-avoid shadow-amber-600 shadow-blocks-sm font-main">
-						<p className="text-base">
-							Use this website to create your passport data page. 🛂
-						</p>
-						<p className="text-base">
-							The info can be whatever you want. Some people have put their real
-							info, others have put totally wacky stuff. Half of the passports
-							have a cat set as their portrait.
-						</p>{" "}
-						{userId ? (
-							<p className="text-base">
-								When you&#39;re ready to register for a passport ceremony, check
-								the checkbox to be assigned a passport number. Your passport
-								will be printed & ready to assemble at the next ceremony.
+					{!latestPassport ? (
+						<>
+							<div className="rounded-sm border-[3px] border-amber-400 flex flex-col justify-center w-full md:w-10/12 gap-2 p-3 sm:p-4 my-4 mx-auto break-inside-avoid shadow-amber-600 shadow-blocks-sm font-main">
+								<p className="text-base">
+									Use this website to create your passport data page. 🛂
+								</p>
+								<p className="text-base">
+									The info can be whatever you want. Some people have put their real
+									info, others have put totally wacky stuff. Half of the passports
+									have a cat set as their portrait.
+								</p>{" "}
+								{userId ? (
+									<p className="text-base">
+										When you&#39;re ready to register for a passport ceremony, check
+										the checkbox to be assigned a passport number. Your passport
+										will be printed & ready to assemble at the next ceremony.
+									</p>
+								) : null}
+							</div>
+						</>
+					) : (
+						<div className="rounded-sm border-[3px] border-amber-400 flex flex-col justify-center w-full md:w-10/12 gap-2 p-3 sm:p-4 my-4 mx-auto break-inside-avoid shadow-amber-600 shadow-blocks-sm font-main">
+							<p>
+								SOMETHING GOES HERE (MAKE IT NOT AMBER TOO) Ceremony Date:{" "}<br />
+								{getCeremonyTimeString(latestPassport.ceremony_time)}<br />
+								This will show for all &quot;existing&quot; passports, so it
+								will be here when returning as well
```

Comment:
Let's do:

You're registered for the passport ceremony happening on <date> at <time>!

Keep it amber

Reply:
Also make sure it doesn't show if the ceremony is in the past or the passport is activated

Reply:
Resolved in [0dd1bd9](https://github.com/purduehackers/passport-issuing-office/pull/52/commits/0dd1bd90777c2dde7ee3c25d4d121a904f74843b)